tests/sentry/workflow_engine/test_task_integration.py

```diff
@@ -0,0 +1,72 @@
+from unittest import mock
+
+from sentry.issues.ingest import hash_fingerprint
+from sentry.issues.occurrence_consumer import _process_message
+from sentry.issues.status_change_consumer import update_status
+from sentry.issues.status_change_message import StatusChangeMessageData
+from sentry.models.group import GroupStatus
+from sentry.models.grouphash import GroupHash
+from sentry.testutils.cases import TestCase
+from sentry.types.activity import ActivityType
+from sentry.types.group import GroupSubStatus
+from tests.sentry.issues.test_status_change_consumer import get_test_message_status_change
+
+
+class IssuePlatformIntegrationTests(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.detector = self.create_detector(project=self.project)
+        self.group = self.create_group(
+            project=self.project,
+            status=GroupStatus.UNRESOLVED,
+            substatus=GroupSubStatus.ESCALATING,
+        )
+        self.fingerprint = f"detector:{self.detector.id}"
+        self.hashed_fingerprint = hash_fingerprint([self.fingerprint])
+        GroupHash.objects.create(
+            project=self.project,
+            hash=self.hashed_fingerprint[0],
+            group=self.group,
+        )
+
+    def test_handler_invoked__when_update_status_called(self):
```

Comment:
Is this the only new test? The other one looks like it was just moved from another file

Reply:
yeah - i added an larger integration test to ensure processing the message called into process_workflow as expected.